app/main.py

```diff
@@ -2,4 +2,68 @@ class Cargo:
     def __init__(self, weight: int) -> None:
         self.weight = weight
 
-# write your code here
+
+class BaseRobot:
+    def __init__(
+            self,
+            name: str,
+            weight: int | float,
+            coords: list = None
+    ) -> None:
+        self.name = name
+        self.weight = weight
+        self.coords = [0, 0] if coords is None else coords
+
+    def go_forward(self, y_coord: int = 1) -> None:
+        self.coords[1] += y_coord
+
+    def go_back(self, y_coord: int = 1) -> None:
+        self.coords[1] -= y_coord
+
+    def go_right(self, x_coord: int = 1) -> None:
+        self.coords[0] += x_coord
+
+    def go_left(self, x_coord: int = 1) -> None:
+        self.coords[0] -= x_coord
+
+    def get_info(self) -> str:
+        return f"Robot: {self.name}, Weight: {self.weight}"
+
+
+class FlyingRobot(BaseRobot):
+    def __init__(
+            self,
+            name: str,
+            weight: int | float,
+            coords: list = None
+    ) -> None:
+        super().__init__(name, weight, coords)
+        self.coords = [0, 0, 0] if coords is None else coords
```

Comment:
You could avoid redefining `self.coords` and pass to the `super().__init__(` the actual value using `or`
```suggestion
        super().__init__(name, weight, coords or [0, 0])
```